integration/spark/src/main/scala/org/carbondata/spark/rdd/CarbonGlobalDictionaryRDD.scala

```
@@ -208,6 +256,7 @@ class CarbonBlockDistinctValuesCombineRDD(
       case ex: Exception =>
         LOGGER.error(ex)
```

Should not catch all exceptions, any executor failures, spark will not relaunch executor